app/src/models/data/IReport.ts

```diff
@@ -2,22 +2,16 @@
 export interface IReport {
```

Comment:
Is there a reason to flatten `contactInformation`, `issueLocation`, etc? I don't have a strong preference either way, but I think it makes sense for these to be grouped :)